src/menu.js

```diff
@@ -0,0 +1,47 @@
+export function loadMenuPage() {
```

Comment:
To much code repeated, try to break this function using some helper functions to avoid repetation.

Reply:
You can reuse these helper function across all .js files, if you extract these utitlity functions to a new file called `utils.js`